querybook/webapp/ui/Modal/FullScreenModal.tsx

```diff
@@ -18,20 +18,20 @@ export const FullScreenModal: React.FunctionComponent<IStandardModalProps> = ({
     });
 
     const modalTitleDOM = (
-        <>
+        <div className="Modal-top horizontal-space-between">
+            {title && <div className="Modal-title">{title}</div>}
```

Comment:
the `title &&` is not needed (since it is checked below `title && modalTitleDOM`